src/main/java/edu/cornell/kfs/concur/batch/service/ConcurRequestExtractReportService.java

```diff
@@ -7,5 +7,8 @@
 public interface ConcurRequestExtractReportService {
     
     File generateReport(ConcurRequestExtractBatchReportData reportData);
-
+    
+    void sendResultsEmail(ConcurRequestExtractBatchReportData reportData, File reportFile);
+    
+    void sendNoReportEmail();
```

Comment:
Could you please rename this method to make the intent a little clearer? Depending on how you read it, someone might think that the method is not sending an email at all.